docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -223,17 +236,20 @@ To enable SNI, use the following property:
 quarkus.tls.key-store.sni=true # Disabled by default
 ----
 
-With this setting enabled, the client indicate the server name during the TLS handshake, allowing the server to select the right certificate:
+With this setting enabled, the client indicates the server name during the TLS handshake, allowing the server to select the appropriate certificate:
 
-- When configuring the keystore with PEM files, multiple CRT/Key must be given.
-- When configuring the keystore with a JKS or a P12 file, it selects one alias based on the SNI hostname. In this case, all the keystore password and alias password must be the same. Do not set the `alias` property in this case.
+* When configuring the keystore with PEM files, multiple certificate (CRT) and key files must be provided.
```

Comment:
It's not clear to me how the server picks the certificate in this case (for JKS/P12 it says that the server selects one alias based on the SNI hostname, but how does it work here?). I'm not that familiar with it, it may need @sberyozkin or @cescoffier

Reply:
It selects the rights certificates/private key based on the host of the request. It then matches with the common name (CN - Deprecated but still widely used)  or the SAN (subject alternative names, heavily recommended) configured in the certificate.